### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#71]
This audit should be, hopefully, pretty straight forward.

### Validation standard
WCAG 2.2 AA

### Not needed
- Detailed write up on issues
- Screenshot unless critical to understand the issue

### Needed
- Review of the following pages:
-- Homepage https://dev.nicolas-steenhout.com/
-- Meet Nic https://dev.nicolas-steenhout.com/meet-nic/
-- Speaking https://dev.nicolas-steenhout.com/speaking/
-- Training https://dev.nicolas-steenhout.com/training/
-- Podcast https://dev.nicolas-steenhout.com/podcast/
-- Contact https://dev.nicolas-steenhout.com/contact/
-- Blog https://dev.nicolas-steenhout.com/blog/
-- 1 blog page at random

### Deliverables
Please deliver results in whichever way is easiest for you. Either a word doc, one issue here with all findings in one, or one issue per finding, assigned to me.

### Known issues
- Some minor layout and styling issues, included on mobile. These should have no impact on accessibility but are being worked on anyway before Go Live
- No caption or transcript on video speaker reel. The video isn't final. Captions and transcripts are going to be added with final version of video.
- Testimonials are not marked up as lists - this is planned work
- Some filler text still exist. This is being worked on. Please disregard any bacon ipsum 
- Sitemap has weird gaps hence empty list items.
